### PR TITLE
Init sig-device-iot info

### PR DESCRIPTION
**Pull request by fisherxu**
Continue of SIG charter draft at https://docs.google.com/document/d/1Ieh-e-4cO_Uvtjv38D45wlafFN4yZvL9zgw353JxYMs/edit

### CODE REVIEW

File: sig-device-iot/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# SIG Device IoT
+
+SIG Device IoT is responsible to simplify, develop, and maintain device (protocols) integrations as extensions, or add-ons, to KubeEdge deployments.
+
+The [charter](./charter.md) defines the scope and governance of the Auth Special Interest Group.
```
Comment (kevin-wangzefeng) [inline sig-device-iot/README.md:5]:
```suggestion
The [charter](./charter.md) defines the scope and governance of the Device IoT Special Interest Group.
```

File: sig-device-iot/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Meetings
+
+* Regular SIG Meeting: [Friday at 10:00 UTC+8](https://zoom.us/j/4167237304) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```
Comment (kevin-wangzefeng) [inline sig-device-iot/README.md:9]:
```suggestion
* Regular SIG Meeting: [Friday at 10:00 UTC+8](https://zoom.us/j/4167237304) (biweekly, starts on Sep. 25th 2020). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
```

File: sig-device-iot/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1PeuZzfLjU8oIduKI2nFeckKAhhckME4tqCZ33WmORBI/edit#heading=h.252sw1n3b1vb).
+ * [Meeting Calendar](https://calendar.google.com/calendar/embed?src=8rjk8o516vfte21qibvlae3lj4%40group.calendar.google.com) | [Subscribe](https://calendar.google.com/calendar?cid=OHJqazhvNTE2dmZ0ZTIxcWlidmxhZTNsajRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+ 
+## Contact
+- Slack: [#sig-node](https://kubeedge.slack.com/archives/C01239D6PM4)
```
Comment (kevin-wangzefeng) [inline sig-device-iot/README.md:14]:
```suggestion
- Slack: [#sig-device-iot](https://kubeedge.slack.com/archives/C01239D6PM4)
```

File: sig-device-iot/charter.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# SIG Device IoT Charter
+
+This charter adheres to the conventions described in KubeEdge Open Governance and uses the Roles and Organization Management outlined in the governance doc.
```
Comment (kevin-wangzefeng) [inline sig-device-iot/charter.md:3]:
```suggestion
This charter adheres to the conventions described in [KubeEdge Open Governance](https://github.com/kubeedge/community/blob/master/GOVERNANCE.md) and uses the Roles and Organization Management outlined in the governance doc.
```

**Comment by chendave** [comment]
/lgtm
Nice initiative!

**Pull request merged**